modules/nw-operator-cr.adoc

```diff
@@ -206,7 +206,7 @@ endif::operator[]
 
 |`gatewayConfig`
 |`object`
-|Optional: Specify a configuration object for customizing how egress traffic is sent to the node gateway.
+|Optional: Specify a configuration object for customizing how egress traffic is sent to the node gateway. Accepted values are `Shared` and `Local`. The default value is `Shared`. In the default setting, all external traffic leaving the cluster is routed through a centralized gateway node and pods use a single egress IP for outbound traffic. In the `Local` setting, external traffic from pods exits through the node that it is running on. Each node serves as its own gateway.
```

Comment:
This is incorrect. In both `Shared` and `Local` gateway modes, the traffic exits from the pod node unless it has to use an egress IP located in a different node.

The difference between `Shared` and `Local` is that, in `Shared`, OVS outputs traffic directly to the node IP interface, while in `Local` it traverses the host network (so it gets applied the routing table of the host, etc.)

Reply:
It might be possible that you are misinterpreting some details on how the **logical** architecture at OVN level used to be described. But here it is best to stick to what happens physically, which is what I described above.

Reply:
Updated